src/main/java/core/selenium/WebDriverManager.java

```diff
@@ -5,31 +5,54 @@
 
 import java.util.concurrent.TimeUnit;
 
+/**
+ * WebDriverManager class, this class is responsible for creating an instance of a browser.
+ *
+ * @author Maday Alcala
+ * @version 0.0.1
+ */
 public final class WebDriverManager {
-    private WebDriverConfig webDriverConfig = WebDriverConfig.getInstance();
-    private WebDriver webDriver;
+    private static WebDriverConfig webDriverConfig = WebDriverConfig.getInstance();
```

Comment:
WebDriverConfig is a singleton not need to declare or instantiate.

Reply:
I'm not really instantiating it, just declare the variable "webDriverConfig" and assign it: "WebDriverConfig.getInstance();", since this sentence is repeated many times, and because it's a little long and repeated more than twice I decided to save it in that variable.

Reply:
Solved.